### PR TITLE
2023 11 13 early access journey

### PR DESCRIPTION
**Pull request by christophercameron-ixd**
# PR Checklist

- [X] Set yourself as the Assignee
- [X] Tag anyone you would like to review, or @forms-design or @forms-devs
- [X] Fill in the template below

---

# What

Added documentation of design work and agreed MVP for the early access journey we are including in the forms platform.

# How to review

1. Do you agree with the write up?
2. Are there any key decisions or agreements we made that have been misinterpreted or missed?
3. Spelling, grammar, etc.

# Who can review

Anyone in the team can review.

### CODE REVIEW

File: design/features/early-access/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<br>
+
+## Problems we tried to solve
+* There is currently no process in place where users can request to become an editor, sign mou and go through requirements check to become an editor.
+* There is also no process in place for adoption to receive requests for editor acccess
+
+<br>
+
+## User needs: 
+ * As a trial user I need to know what are the criteria to become an editor
```
Comment (bettymw) [inline design/features/early-access/README.md:37]: Made an edit to this user need 

"As a trial user I need to know what are the criteria and steps to become an editor"

**Review by bettymw — DISMISSED** [review]
Looks good to me, made just one small comment on the user needs.

**Review by DavidBiddle — DISMISSED** [review]

**Review by hannahkc — DISMISSED** [review]
Made some changes. All good now. Probably needs a good review from @hazalarpalikli

**Comment by hazalarpalikli** [comment]
This looks good to me too! I made minor changes as i spotted some typo. It's asking everyone to review it again. Is there a way we can bypass this and just close this PR?

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**